docker/Dockerfile-dev

```diff
@@ -1,5 +1,8 @@
 FROM python:3.5-alpine
 
+# TODO: Does this need to be done on Dockerfile-prod?
```

Comment:
I added a [card](https://trello.com/c/k7OkjSJf/352-2-update-ui-dockerfile-prod-runtime-dependencies) for this.